code/game/objects/structures/tables_racks.dm

```diff
@@ -45,15 +45,28 @@
 
 /obj/structure/table/ex_act(severity, target)
 	..()
-	if(severity == 3)
-		if(prob(25))
-			table_destroy(1)
+	switch(severity)
+		if(1)
+			qdel(src)
+		if(2)
+			if(prob(50))
+				table_destroy(1)
+			else
+				qdel(src)

```

Comment:
Severities 1 & 2 are already handled in the parent. Don't call parent if you're gonna do this